service/controller/prometheus.go

```diff
@@ -92,7 +92,7 @@ func NewPrometheus(config PrometheusConfig) (*Prometheus, error) {
 			ResourceSets: []*controller.ResourceSet{
 				resourceSetV1,
 			},
-			RESTClient: config.K8sClient.CoreV1().RESTClient(),
+			RESTClient: config.K8sClient.RESTClient(),
```

Comment:
Yes I think makes sense to use the configured REST client.

Reply:
it's getting an error from this line, the issue described in https://github.com/giantswarm/g8s-prometheus/pull/660